.travis.yml

```diff
@@ -1,11 +1,6 @@
 language: ruby
 rvm:
-  - 2.0.0
-  - 2.1.10
-  - 2.2.7
-  - 2.3.5
-  - 2.4.1
+  - 2.6.5
```

Comment:
@jbusser This is a general question, not specific to this PR or project -- what are your thoughts on specifying only the minor version in Travis config (ie. `2.6` vs `2.6.5`)?

Reply:
I never really thought about it. What opinions do you have? What are other projects doing?

Reply:
We have variance :) among the projects that use Travis. I think using minor versions eases the maintenance of patch level versions in the config, especially if the project is scheduled to build regularly.